libraries/AP_Vehicle/AP_Vehicle.h

```diff
@@ -170,10 +170,12 @@ class AP_Vehicle : public AP_HAL::HAL::Callbacks {
     virtual bool start_takeoff(float alt) { return false; }
     virtual bool set_target_location(const Location& target_loc) { return false; }
     virtual bool set_target_velocity_NED(const Vector3f& vel_ned) { return false; }
+    virtual bool set_target_angle(const float roll_deg, const float pitch_deg, const float yaw_deg, const float climb_rate_cms, const bool use_yaw_rate, const float yaw_rate_rads) { return false; }
```

Comment:
instead of "set_target_angle" it might be "set_target_angle_and_climbrate" but again not a blocker.

Reply:
fixed